js/main.js

```diff
@@ -23,6 +23,9 @@ function fetchData () {
     })
     .then(res => res.json())
     .then(res => {
+        if(res.incompleteUserDetails){
+            return window.location.replace('https://my.realdevsquad.com/signup')
+        }
```

Comment:
Wouldn't it be better to alert the user where the user is now being taken to? We could show an alert saying "You'll now be redirected to a form where you can complete your profile"
What do you think? 🤔

Reply:
Ya we can. The only thing is alert will stop the redirection and ask for the user's response to make the redirection happen. The user may bounce back and probably will not do the signup. 
The way right now, user will simply redirect to signup post the github auth. S M O O T H ................

Reply:
This is good per discussion with @ankushdharkar